packages/cli/src/ui/App.tsx

```diff
@@ -454,7 +465,24 @@ const App = ({ config, settings, startupWarnings = [], version }: AppProps) => {
       setConstrainHeight(true);
     }
 
-    if (key.ctrl && input === 'o') {
+    if (key.escape) {
```

Comment:
can you move this escape logic into InputPrompt rather than having App.tsx deal with functionality that is focused on input prompt?